app/src/main/java/zapsolutions/zap/util/Wallet.java

```diff
@@ -155,6 +160,7 @@ public void isLNDReachable() {
 
             LightningGrpc.LightningFutureStub asyncInfoClient = LightningGrpc
```

Comment:
Not particular a thing for this PR, just a question that came to my mind: I was wondering if we could keep this client as a member and use it multiple times?

Reply:
I tried to reuse one LightningGrpc.LightningFutureStub for all async rpc calls, but that crashed the app. The blockingStub can be reused. We might give it another try, though.